.github/workflows/build-pr-development.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 8, 11 ]
+        java: [ 11 ]
```

Comment:
@gsmet is this okay? Quarkus no longer seems to build with 8: https://github.com/quarkusio/quarkus-quickstarts/pull/829/checks?check_run_id=2341866380

Reply:
I see the link redirects now to a newer test run. Building Quarkus failed, thje error message was about the `--release` flag.

Reply:
here's the failure for another quickstart I'm adding: https://github.com/quarkusio/quarkus-quickstarts/pull/832/checks?check_run_id=2342157467